### ISSUE TITLE
Namespaced operations.

### ISSUE THREAD

**Issue by scottjbarr** [fiorix/wsdl2go#9]
Some endpoints require namespaced messages, for example...

```
<SOAP-ENV:Envelope 
    xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
    xmlns:ns="http://example.com/WatWebServiceAPI?wsdl">
    <SOAP-ENV:Body>
        <mns1:watsay xmlns:mns1="http://WatWebServiceAPI/">
            <somevalue>WAT</somevalue>
        </mns1:watsay>
    </SOAP-ENV:Body>
</SOAP-ENV:Envelope>
```

instead of (note the `watsay` entity)

```
<SOAP-ENV:Envelope 
    xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
    xmlns:ns="http://example.com/WatWebServiceAPI?wsdl">
    <SOAP-ENV:Body>
        <watsay xmlns="http://WatWebServiceAPI/">
            <somevalue>WAT</somevalue>
        </watsay>
    </SOAP-ENV:Body>
</SOAP-ENV:Envelope>
```

This seems to be the case when the WSDL specifies the `targetNamespace` and `xmlns:tns` in the `wsdl:definitions` entity. I may be wrong on the actual reason for this.

Example snippet from WSDL.

```
    <?xml version="1.0" encoding="utf-8"?>
    <wsdl:definitions 
            xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" 
            xmlns:tns="WatWebServiceAPI/" 
            xmlns:xs="http://www.w3.org/2001/XMLSchema" 
            xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" 
            targetNamespace="WatWebServiceAPI/" 
            name="WatService">
        <wsdl:types>
        ...
```

**Comment by fiorix** [comment]
I'm not sure we can support this entirely because of how `encoding/xml` works. There's more explanations here: https://github.com/golang/go/issues/14407

**Comment by fiorix** [comment]
Likely fixed with #75. Happy to reopen this if needed.
